app/models/art_piece.rb

```diff
@@ -68,4 +68,26 @@ def by_student?
     artists.find{|x| x.student?}.present?
   end
 
+  def self.to_csv
+    CSV.generate do |csv|
+      csv << ["Art Piece Name", "Created on", "Size", "Legal Info", "Contact Info", "Description", 
+              "Displayed", "Private", "Number", "Artist Comments", "Art Piece Latitude", 
+              "Art Pieve Longitude", "Photo Credit", "Artist Name", "Artist Bio", "Artist Birthdate",
```

Comment:
I was thinking something like:

art.art_piece_photos.map{|photo| photo.credit}.compact.join(", ")

just so that if there are different credits that are important, they are all recognized.